### PR TITLE
feat(inverse): add builder API

### LINKED ISSUE #12: Use a builder API for inverse derivatives
Following the example set in #10

### PR DESCRIPTION
**Pull request by EricCrosson**
Follows the example set in #10

Closes #12

### CODE REVIEW

File: src/inverse/ws.rs [hunk 1]
```diff
@@ -9,6 +9,9 @@ use std::time::Duration;
 use tungstenite::{connect, Message};
 use url::Url;
 
+const MAINNET: &str = "wss://stream.bybit.com/realtime";
+const TESTNET: &str = "wss://stream-testnet.bybit.com/realtime";
```
Comment (EricCrosson) [inline src/inverse/ws.rs:13]: Thankfully, there don't seem to be as many choices for inverse derivatives as there are for the spot websockets

**Comment by jukanntenn** [comment]
I leave a comment in #13 for disussing about chosing between &str and AsRef<str> generic for parameters with natural string type. It is also related to this PR. Please take a look.

**Comment by EricCrosson** [comment]
Hi @jukanntenn, I don't see a comment on #13 yet, I'll take a look when it is published :slightly_smiling_face:

**Comment by jukanntenn** [comment]
Sorry, I forgot publish ⊙﹏⊙|||. It is published now.

**Pull request merged**